testsuite/compare.cmake

```diff
@@ -0,0 +1,27 @@
+# This is used to compare to files using CMake.
+# It improves on the default behaviour to print the
+# the files if there is a difference.
+
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -E compare_files --ignore-eol ${original_file} ${new_file}
+    RESULT_VARIABLE status
+)
+
+if (${status} EQUAL 1)

```

Comment:
Is this the actual return status or some CMake magic? If return status, non-zero can be anything, not just 1.

Reply:
Actual return status,  unless cmake fails for some reason (e.g. TIMEOUT, can't find the command), and then it will contain an error message explaining the failure.

Reply:
Ah, sorry, this is just the compare command, not the program exit status. Ignore me.